features/index.feature

```diff
@@ -0,0 +1,44 @@
+Feature:
+   As a User|
+   In order to navigate index.erb
+   I want to see a menu
+
+   Backgroud:
+   Given that this is the first page user arives to, there are no previous page.
+
+
+
+
+
+   Scenario: User arives to page
+   When I visit the site
```

Comment:
Because you are doing `When I visit the site` in all of these scenarios, you can just put it into a `Background` step - then it will happen for every scenario.